website/report.20190906.145200.23403.0.001.json

```diff
@@ -0,0 +1,552 @@
+{
```

Comment:
this file doesn't look like it should be committed?

Reply:
Yeah I agree. @raechelo could you add `./website/report.*` to the `.gitignore` file in the repo and commit your changes?